pkgs/top-level/python-packages.nix

```diff
@@ -25797,4 +25797,21 @@ in modules // {
     };
   };
 
+  topydo = buildPythonPackage rec {
+    name = "topydo-${version}";
+    version = "0.9";
+    disabled = (!isPy3k);
+
+    src = pkgs.fetchFromGitHub {
```

Comment:
if there is a pypi package available, use that one.
https://pypi.python.org/pypi/topydo/0.9